BfA/Freehold/Locales/esES.lua

```diff
@@ -34,15 +34,15 @@ end
 L = BigWigs:NewBossLocale("Ring of Booty", "esES") or BigWigs:NewBossLocale("Ring of Booty", "esMX")
 if L then
 	-- Gather 'round and place yer bets! We got a new set of vict-- uh... competitors! Take it away, Gurgthok and Wodin!
-	--L.lightning_warmup = "new set of vict--"
+	L.lightning_warmup = "nuevo grupo de víct--"
 	-- It's a greased up pig? I'm beginning to think this is not a professional setup. Oh well... grab the pig and you win
-	--L.lightning_warmup_2 = "not a professional setup"
+	L.lightning_warmup_2 = "no es muy profesional"
```

Comment:
With these `_warmup` translations and the other `_trigger` translations such as `L.death_pact_trigger` and `L.avanoxx_warmup_trigger` below, the translated text must match exactly what the NPCs say in-game or they won't function properly.

Did you gather these translations from the associated dungeons?

Reply:
If these are the exact dialogue lines the NPC says in the game in Spanish, it’s better to leave them as they are to avoid errors, since they barely interfere. I translated those, but then I didn’t continue with the other dungeons and didn’t revert those back to their English version.